reference/tools/gnu/pkgconfigdeps.rst

```diff
@@ -111,6 +111,8 @@ you can activate it using the **build_context_activated** attribute:
 build_context_suffix
 ^^^^^^^^^^^^^^^^^^^^
 
+*DEPRECATED: use build_context_folder attribute instead*
```

Comment:
Is this true for ``autotools`` projects too? Or only for ``Meson``? I think only Meson has the capability of specifying a folder for the build-requires?